x-pack/plugins/ml/public/explorer/components/explorer_no_influencers_found/explorer_no_influencers_found.js

```diff
@@ -16,7 +16,6 @@ import { EuiEmptyPrompt } from '@elastic/eui';
 
 export const ExplorerNoInfluencersFound = ({ swimlaneViewByFieldName, showFilterMessage = false }) => (
   <EuiEmptyPrompt
-    iconType="alert"
```

Comment:
Not specifically related to this PR, but in the case where there are no influencers returned for the selected time and filter, but there are influencer results for the job, if we should change the warning text to something like `No <fieldname> influencers found matching filter`?